### PR TITLE
Delete hall events

### LINKED ISSUE #92: Create command to delete event
User story: As a hall admin, I can delete hall events.

### PR DESCRIPTION
**Pull request by jieffyy**
Add a command to delete hall-events.

To be reviewed/merged after PR 115.

Closes #92.

Will add test cases in another PR.

image

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/DeleteEventCommand.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+public class DeleteEventCommand extends Command {
+
+    public static final String COMMAND_WORD = "delete-event";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Deletes the event identified by the index number used in the displayed event list.\n"
+            + "Parameter: INDEX (must be a positive integer)\n"
+            + "Example: " + COMMAND_WORD + " 1";
+
+    public static final String MESSAGE_DELETE_PERSON_SUCCESS = "Deleted Event:\n%1$s";
```
Comment (teekoksiang) [inline src/main/java/seedu/address/logic/commands/DeleteEventCommand.java]: Hmm you mean `MESSAGE_DELETE_EVENT_SUCCESS`?

**Review by teekoksiang — COMMENTED** [review]
Nice work, almost LGTM! Just a minor nit for the naming of a variable.

File: src/main/java/seedu/address/logic/commands/DeleteEventCommand.java (hunk 1 above)
Comment (jieffyy) [reply src/main/java/seedu/address/logic/commands/DeleteEventCommand.java]: Good catch! Fixed

**Review by teekoksiang — APPROVED** [review]
LGTM

**Pull request merged**